lib/converters/v1.0.0/converter-v1-to-v2.js

```diff
@@ -210,8 +210,10 @@ _.assign(Builders.prototype, {
      * @returns {Array}
      */
     event: function (entityV1) {
+        if (!entityV1) { return; }
+
         // events is treated as the source of truth in v1, so handle that first and bail out.
-        if (_.isArray(entityV1.events)) {
+        if (util.notLegacy(entityV1, 'script') && _.isArray(entityV1.events)) {
```

Comment:
this should be called `util.notLegacy(entityV1, 'event')`. we may have to do script transformations later.